tools/azure-sdk-tools/packaging_tools/sdk_generator.py

```diff
@@ -273,23 +273,22 @@ def main(generate_input, generate_output):
     result = {}
     python_tag = data.get("python_tag")
     package_total = set()
-    spec_word = "readmeMd"
+    readme_files = []
     if "relatedReadmeMdFiles" in data:
-        readme_files = data["relatedReadmeMdFiles"]
-    elif "relatedReadmeMdFile" in data:
+        readme_files.extend(data["relatedReadmeMdFiles"])
+    if "relatedReadmeMdFile" in data:
         input_readme = data["relatedReadmeMdFile"]
         if "specification" in spec_folder:
             spec_folder = str(Path(spec_folder.split("specification")[0]))
         if "specification" not in input_readme:
             input_readme = str(Path("specification") / input_readme)
-        readme_files = [input_readme]
-    else:
+        readme_files.append(input_readme)
+    if "relatedTypeSpecProjectFolder" in data:
         # ["specification/confidentialledger/ConfientialLedger"]
         if isinstance(data["relatedTypeSpecProjectFolder"], str):
```

Comment:
`data["relatedTypeSpecProjectFolder"]` should be list type instead of string type?

Reply:
It is compatible logic added by dongwei before and I am not sure whether now the input data will never be string anymore.

Reply:
It's always a string array as the following schema. BTW, `relatedReadmeMdFile` was deprecated and can be deleted.
```typescript
 "relatedReadmeMdFiles": {
      // Related readme.md files that pending generation.
      "type": "array",
      "items": {
        "type": "string"
      }
    },
    "relatedTypeSpecProjectFolder": {
      // Related typespec project folder that pending generation.
      "type": "array",
      "items": {
        "type": "string"
      }
    },
```

Reply:
Updated.